clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/ReadGroupWebService.scala

```diff
@@ -29,12 +31,10 @@ trait ReadGroupWebService {
 
   private[webservice] val postMetadata: Route = {
     pathPrefix("metadata") {
-      pathPrefix("v1") {
-        pathPrefixKeyV1 { key =>
-          post {
-            entity(as[TransferReadGroupV1Metadata]) { metadata =>
-              complete(readGroupService.upsertMetadata(key, metadata))
-            }
+      pathPrefixKeyV1 { key =>
```

Comment:
I vacillated on both of these comments.  It is still a V1 key, matching up with the V1 transfer cases. We could build the routes up in the server like I did in the Python test, but I think it is easier to read the tests with the explicit subroutes. But I don't have strong opinions either way unless I missed your point.